cdn-examples/server/auth0/DX/index.js

```diff
@@ -0,0 +1,480 @@
+/* eslint-disable no-undef */
+/* eslint-disable no-restricted-globals */
+/**
+ * Example code for a Cloudflare Worker that can proxy Auth0 requests and inject an
+ * Arkose Labs integration into and then process the Arkose Labs result on the
+ * Cloudflare CDN layer with Data Exchange.
+ *
+ * This requires the following environment variables being setup for the Cloudflare Worker
+ * @param {string} publicKey The Arkose Labs public key to use
+ * @param {string} privateKey The Arkose Labs private key to use for verification
+ * @param {string} clientSubdomain A customer's specific subdomain used for the loading of the Client-API (if setup)
+ * @param {string} verifySubdomain A customer's specific subdomain used for the verification call (if setup)
+ * @param {string} errorUrl A url to redirect to if there has been an error
+ * @param {string} cookieName The name of the cookie to store the Arkose Labs session token in
+ * @param {string} failOpen A boolean string to indicate if the current session should fail
```

Comment:
Some of these are typed incorrectly, ie failOpen is a boolean but set to a string